src/utils_request.js

```diff
@@ -16,9 +16,11 @@ export const REQUEST_AS_BROWSER_DEFAULT_OPTIONS = {
     json: false,
     abortFunction: (res) => {
         const { type } = contentType.parse(res.headers['content-type']);
-        return res.statusCode === 406 || type.toLowerCase() !== 'text/html';
+        return res.statusCode === 406 || type.toLowerCase() !== 'text/html' || res.statusCode >= 500;
     },
     useCaseSensitiveHeaders: true,
+    useStream: false,
+    payload: '',
```

Comment:
Either use `null` or nothing. `''` looks like something that will cause a lot of bugs.